doc(release) Release notes collect 24.04.15 gorgone 24.04.11 (#4690)
centreon/centreon-documentation: versioned_docs/version-24.04/releases/centreon-os.mdx

## Centreon Collect

### 24.04.15

Release date: `September 29, 2025`

<details>
  <summary>Bug fixes</summary>

> - [Broker] Fixed a segfault on Broker that was occuring when Engine stopped or was restarted, when Engine's version was older than 24.04.

Reviewer: ```suggestion
- [Broker] Fixed a segfault on Broker that was occurring when Engine stopped or was restarted, when Engine's version was older than 24.04.
```

> - [Core] Fixed compatibility of engine with older versions of Linux kernel.

Reviewer: ```suggestion
- [Core] Fixed the compatibility of Engine with older versions of the Linux kernel.
```

> - [Engine] Fixed Engine crash when deleting a host during downtime notification.

Reviewer: ```suggestion
- [Engine] Engine no longer crashes when users delete a host during downtime notification.
```

## Centreon Gorgone

### 24.04.11

Release date: `September 29, 2025`

<details>
  <summary>Bug fixes</summary>

> - [Gorgone] Fixed centreontrapd configuration file permissions when exporting poller configuration.

Reviewer: ```suggestion
- [Gorgone] Fixed centreontrapd configuration file permissions applied when exporting poller configuration.
```

> - [Gorgone] Raise an error if ETL configuration file don't exist.

Reviewer: ```suggestion
- [Gorgone] Gorgone now raises an error if the ETL configuration file doesn't exist.
```

> - [MBI] Fixed a bug in MBI where hostname with semicolon will break database insertion.	

Reviewer: ```suggestion
- [MBI] Fixed a bug in MBI where a hostname with a semicolon broke database insertion.	
```

> - [MBI] Fixed an issue where ETL would never finish if some service templates were referencing themselfes.

Reviewer: ```suggestion
- [MBI] Fixed an issue where the ETL would never finish executing if some service templates were referencing themselves.
```

</details>

<details>
  <summary>Security fixes</summary>

> - [Gorgone] Added option to make discovery commands less permissive.

Reviewer: ```suggestion
- [Gorgone] Added an option to make discovery commands less permissive.
```
